lib/rspec/bash/matchers/called_with_arguments.rb

```diff
@@ -16,4 +17,14 @@
 
     called_with_correct_args && called_correct_number_of_times
   end
+
+  failure_message do |actual_command|
+    "expected #{actual_command.command} to be called with arguments #{expected_argument_list}"
```

Comment:
this prints out some really ugly output when using anything matchers, we can address that later?